metricq/history_client.py

```diff
@@ -305,14 +362,18 @@ async def history_last_value(self, metric: str, timeout=60):
         return next(result.values())
 
     async def history_metric_list(self, selector=None, historic=True, timeout=None):
-        """
-        DEPRECATED! use get_metrics
-        """
-        return await self.get_metrics(
-            selector=selector, historic=historic, timeout=timeout
-        )
+        with warnings.catch_warnings():
+            warnings.simplefilter("always", category=DeprecationWarning)
+            warnings.warn(
+                f"history_metric_list() is deprecated: use {type(self).__name__}.get_metrics()",
+                DeprecationWarning,
+            )
+            return await self.get_metrics(
+                selector=selector, historic=historic, timeout=timeout
+            )
 
     async def history_metric_metadata(self, selector=None, historic=True):
+        # TODO: should this be deprecated in favour of get_metrics(..., metadata=True)?
```

Comment:
Don't add todos. Either fix it now or create an issue.

@tilsche What do you think?

Reply:
Fixed with 00cdcfc.